pkg/cvo/updatepayload.go

```diff
@@ -315,3 +317,15 @@ func findUpdateFromConfigVersion(config *configv1.ClusterVersion, version string
 	}
 	return configv1.Update{}, false
 }
+
+func (optr *Operator) clusterProfileRetriever() (string, error) {
+	configMap, err := optr.cmConfigLister.Get(internal.ClusterProfileConfigMap)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			klog.Infof("Cannot get ConfigMap %s/%s, using default profile: %v", internal.ConfigNamespace, internal.ClusterProfileConfigMap, err)
```

Comment:
No need to format `err` in here.  We already know it's not-found and we can express that in English like `ConfigMap %s/%s not found, excluding all profiles`.

Reply:
done